.rustfmt.toml

```diff
@@ -0,0 +1 @@
+
```

Comment:
Accidentally added?

Reply:
Intentional. Otherwise my custom config is applied, and completely changes formatting of this project.

Reply:
Ok. Can you add a comment there that says the intention is to override system-wide rustfmt config?